graphql_compiler/tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018-present Kensho Technologies, LLC.
+
```

Comment:
consistency: the other files have the first import immediately below the copyright line, but this one has a blank line in between